commisery/commit.py

```diff
@@ -97,20 +97,20 @@ def __str__(self):
 class CommitMessage:
     """Conventional Commit Message"""
 
-    body: Optional[Sequence[str]] = field(default_factory=list)
-    breaking_change: str = None
-    description: str = None
-    footers: Optional[Sequence[Footer]] = field(default_factory=list)
-    hexsha: str = None
-    separator: str = None
+    body: Sequence[str] = field(default_factory=list)
+    breaking_change: Optional[str] = None
+    description: Optional[str] = None
+    footers: Sequence[Footer] = field(default_factory=list)
+    hexsha: Optional[str] = None
+    separator: Optional[str] = None
     scope: Optional[str] = None
     type: Optional[str] = None
 
     @property
     def subject(self):
         """Composes the subject line from the provided elements"""
 
-        subject = f"{self.type}"
+        subject = f"{self.type}" if self.type else ""
```

Comment:
Because `self.type` is guaranteed to be of type `str` when it's trueish lets avoid the f-string conversion to string...

```suggestion
        subject = self.type or ""
```